dawn/main/ansible/Ansible.js

```diff
@@ -98,14 +98,17 @@ ipcMain.on('NOTIFY_UPLOAD', () => {
 function buildProto(data) {
   let status = null;
   switch (data.studentCodeStatus) {
-    case stateEnum.TELEOP:
+    case robotState.TELEOP: {
```

Comment:
brackets don't match style in the entire rest of dawn, and these blocks aren't at the level where they would be useful.